lib/app/dependencies.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:dio/dio.dart';
+import 'package:get_it/get_it.dart';
+import 'package:qr_api/qr_api.dart';
+
+GetIt getIt = GetIt.instance;
```

Comment:
Past time ago I take a look into this [issue](https://github.com/fluttercommunity/get_it/issues/5) because I had the same doubt after that I always define it as a global variable. Anyway, both are pretty similar approaches.